### PR TITLE
Drop confusing non-sequitur from addEventListener() doc

### LINKED ISSUE #12500: Issue with "EventTarget.addEventListener()": removeEventListener line of text should be removed
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

#### What information was incorrect, unhelpful, or incomplete?
the removeEventListener below the introductory  paragraph is confusing and unnecessary and should be removed

#### Specific section or headline?
the line of text "They do not need to be removed manually with removeEventListener()." on it's own line of text

#### What did you expect to see?
removeAddListener description should only be mentioned later on, or otherwise the paragraph that precedes it should include this text as "…for this target, they are not added a second time and they do not need to be removed" to clear up the confusion

#### Did you test this? If so, how?
i'm following the tutorials from a fresh set of eyes, and this line threw me.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/eventtarget/addeventlistener`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/eventtarget/addeventlistener/index.md
* Last commit: https://github.com/mdn/content/commit/2bef9a5f18175148702ab2bb3ef5512205eca6da
* Document last modified: 2022-01-19T12:35:54.000Z

</details>

### PR DESCRIPTION
**Pull request by sideshowbarker**
Fixes https://github.com/mdn/content/issues/12500

This change also makes some other refinements and fixes to the opening part of the addEventListener() doc.

### CODE REVIEW

File: files/en-us/web/api/eventtarget/addeventlistener/index.md [hunk 1]
```diff
@@ -24,20 +24,18 @@ but the target may be any object that supports events (such as {{domxref("XMLHtt
...
 > Indeed, anonymous functions are not identical even if defined using
 > the _same_ unchanging source-code called repeatedly, **even if in a loop**.
 >
 > Repeatedly defining the same unnamed function in such cases can be
 > problematic. (See [Memory issues](#memory_issues), below.)
 
-If an event listener is added to an {{domxref("EventTarget")}} from inside another listener,
-that is during the processing of the event,
+If an event listener is added to an {{domxref("EventTarget")}} from inside another listener —
+that is during the processing of the event —
```
Comment (wbamberg) [inline files/en-us/web/api/eventtarget/addeventlistener/index.md:38]:
```suggestion
that is, during the processing of the event —
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
One copy-edit suggestion, otherwise this looks good.

The page could do with a real overhaul, but that's out of scope for this PR, which is clearly an improvement.

File: files/en-us/web/api/eventtarget/addeventlistener/index.md (hunk 1 above)
Comment (sideshowbarker) [reply files/en-us/web/api/eventtarget/addeventlistener/index.md:38]: 👍 Thanks

**Comment by sideshowbarker** [comment]
> The page could do with a real overhaul

Agreed — because of that, I tried not to read to much of it 😆 (out of fear I’d just fine more threads to start pulling at)

**Review by wbamberg — APPROVED** [review]
👍 thanks for the fix!

**Pull request merged**